docker/android-system.sh

```diff
@@ -116,49 +559,264 @@ EOF
     rm bionic/tests/headers/Android.bp
     rm bionic/tests/headers/posix/Android.bp
 
-    sed -i -z -e 's/cc_test {.*}//g' bionic/libc/malloc_debug/Android.bp
-    sed -i -z -e 's/cc_test {.*}//g' bionic/libc/malloc_hooks/Android.bp
-    sed -i -z -e 's/cc_test_host {.*}//g' bionic/tools/relocation_packer/Android.bp
+    remove_tests
+}
 
-    export ALLOW_MISSING_DEPENDENCIES=true
+# tested on 10.0.0_r47 (SDK 29)
+android_repo_v10() {
+    sync art
+    sync bionic
+    sync build/blueprint
+    sync build/make
+    sync build/soong
+    sync external/clang
+    sync external/compiler-rt
+    sync external/elfutils
+    sync external/golang-protobuf
+    sync external/jemalloc
+    sync external/jemalloc_new
+    sync external/libcxx
+    sync external/libcxxabi
+    sync external/libunwind
+    sync external/libunwind_llvm
+    sync external/llvm
+    sync external/lzma
+    sync external/mksh
+    sync external/vixl
+    sync external/zlib
+    sync libnativehelper
+    sync prebuilts/build-tools
+    sync prebuilts/clang-tools
+    sync prebuilts/clang/host/linux-x86
+    sync prebuilts/gcc/linux-x86/host/x86_64-linux-glibc2.17-4.8
+    sync prebuilts/go/linux-x86
+    sync prebuilts/ndk
+    sync prebuilts/sdk
+    sync prebuilts/vndk/v28
+    sync system/core
+    sync system/sepolicy
 
-    # patch the linker to avoid the error
-    # FATAL: kernel did not supply AT_SECURE
-    #sed -i -e 's/if (!kernel_supplied_AT_SECURE)/if (false)/g' bionic/linker/linker_environ.cpp
```

Comment:
This is only relevant for Android 5, so it's been removed from more recent versions. Since it's documented there, there's no reason to duplicate in every other Android level.